### DISCUSSION TITLE
Multiple paths on `type -a`

### DISCUSSION THREAD

**Discussion by igroti** [Homebrew/discussions#4434] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.14
ORIGIN: https://github.com/Homebrew/brew
HEAD: ac0663ae5d2d4c76bb62363592d77dbad49ac30c
Last commit: 2 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 17eb27263f58d670023fbff983414164ead54a18
Core tap last commit: 5 weeks ago
Core tap branch: master
Core tap JSON: 18 Apr 01:49 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: subl -w
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.87.0 => /usr/bin/curl
macOS: 13.3.1-arm64
CLT: 14.3.0.0.1.1679647830
Xcode: 14.3
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Running `type -a brew` returns:

```
brew is /opt/homebrew/bin/brew
brew is /usr/local/bin/brew
brew is /usr/local/bin/brew
```

Is this OK? Should it not just be homebrew? (M1 Mac)

**Comment by carlocab** [comment]
What's the output of
```
echo $PATH
```
?

**Reply by igroti** [reply]
/opt/homebrew/bin
/Users/dani/Documents/repos/dotfiles
/usr/bin
/bin
/usr/sbin
/sbin
/opt/homebrew/sbin
/usr/local/bin
/usr/local/sbin
/usr/local/bin
/System/Cryptexes/App/usr/bin
/usr/bin
/bin
/usr/sbin
/sbin
/Library/Apple/usr/bin
/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin
/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin
/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
